Source/ReferenceConnectorService/ReferenceConnectorService.csproj

```diff
@@ -15,7 +15,7 @@
     <PackageReference Include="Microsoft.ServiceFabric" Version="6.2.301" />
     <PackageReference Include="Microsoft.ServiceFabric.AspNetCore.Kestrel" Version="3.1.301" />
     <PackageReference Include="Microsoft.ServiceFabric.Services.Remoting" Version="3.1.301" />
-    <PackageReference Include="RecordPoint.Connectors.SDK" Version="0.0.0-CI-20181011-002554" />
+    <PackageReference Include="RecordPoint.Connectors.SDK" Version="0.0.0-CI-20190103-002229" />

```

Comment:
This version of the SDK isn't published on NuGet, so this won't compile outside of RecordPoint. I can publish that version if we need to

Reply:
As there's a new version likely to arrive shortly, I will update this once the new version is done and published.